sound/soc/codecs/mt6357.c

```diff
@@ -4172,6 +4277,12 @@ static int Lineout_PGAR_Set(struct snd_kcontrol *kcontrol,
 		index = DL_GAIN_N_40DB;
 	Ana_Set_Reg(ZCD_CON1, index << 7, 0x0f80);
 	mCodec_data->mAudio_Ana_Volume[AUDIO_ANALOG_VOLUME_LINEOUTR] = index;
+	//Fix first sound loud issue after system boot up finished
+	if (first_call == false) {
+	  Speaker_Amp_Change(1);
+	  first_call = true;
```

Comment:
This seems to turn on the DAC at first boot. Not sure how `Lineout_PGAR_Set` gets called at first boot